common/test/acceptance/pages/studio/overview.py

```diff
@@ -845,6 +845,15 @@ def deprecated_advance_modules(self):
         return self.q(css='.advance-modules-list li').text
 
 
+class CourseOutlineSignInRedirectPage(CourseOutlinePage):
```

Comment:
This should inherit from LoginPage instead of from CourseOutlinePage.
That way if the way that the user logs in is redesigned then we will only need to update one place.
Your call if the is_browser_on_page method of LoginPage should be changed from .present to .visible.